### PR TITLE
Infrastructure for new upload mechanism

### PR DESCRIPTION
**Pull request by wd15**
Address #1299

  - Allow python-pfhub to read directly from meta.yaml files anywhere

**Remember to tear down the live site when merging or closing this
pull request.**

---
This change is [Reviewable](https://reviewable.io/reviews/usnistgov/pfhub/1320)

### CODE REVIEW

**Comment by pfhub** [comment]
The new [PFHub live website](https://random-cat-1320.surge.sh) is ready for review.

**Comment by pfhub** [comment]
The new [PFHub live website](https://random-cat-1320.surge.sh) is ready for review.

**Comment by pfhub** [comment]
The new [PFHub live website](https://random-cat-1320.surge.sh) is ready for review.

File: _data/simulation_list.yaml [hunk 1]
```diff
@@ -0,0 +1,139 @@
+- https://raw.githubusercontent.com/wd15/pfhub/bm8-notebook-comparison-page/_data/simulations/fipy_8a_fake/meta.yaml
```
Comment (tkphd) [inline _data/simulation_list.yaml:1]: YAML documents begin with `---`

**Comment by pfhub** [comment]
The new [PFHub live website](https://random-cat-1320.surge.sh) is ready for review.

**Review by tkphd — APPROVED** [review]
LGTM!

**Comment by tkphd** [comment]
CI faliures:

1. Coverage is not 100%
    ```output
    ----------- coverage: platform linux, python 3.8.9-final-0 -----------
    Name                Stmts   Miss  Cover   Missing
    -------------------------------------------------
    pfhub/conftest.py      30      0   100%
    pfhub/main.py         127      1    99%   109
    -------------------------------------------------
    TOTAL                 157      1    99%

    FAIL Required test coverage of 100% not reached. Total coverage: 99.36%
    ```
2. Broken links in DEVELOPMENT.md:
    * <https://guides.github.com/activities/forking/>
    * <https://help.github.com/articles/creating-a-pull-request/>
    * <https://help.github.com/articles/editing-files-in-another-user-s-repository/>
    * <https://help.github.com/articles/pushing-to-a-remote/>
3. Broken link in     simulations/upload.md:
    * <https://help.github.com/articles/fork-a-repo/>
4. test.py/test_csv():
    ```output
    AssertionError: assert False
    where False = pipe('https://drive.google.com/file/d/1F2Pzo2IYYPhPmU_mryjR6flz2vUDr5Zy/view?usp=sharing',
                       get, <function test_csv.<locals>.<lambda> at 0x7efd5de3a4d0>)
    ```

**Comment by tkphd** [comment]
Opened #1328 to address links: unrelated to this PR. Merge at will.

**Pull request merged**